co-simulation/bundle/src/assembly/resources/etc/runtime.json

```diff
@@ -121,6 +121,21 @@
             ],
             "javaClasspathEntries": []
         },
+        {
+            "id": "carma-cloud",
+            "classname": "org.eclipse.mosaic.fed.carmacloud.ambassador.CarmaCloudMessageAmbassador",
+            "configuration": "carma-cloud_config.json",
+            "priority": 50,
+            "host": "local",
+            "port": 0,
+            "deploy": false,
+            "start": false,
+            "subscriptions": [
+                "V2xMessageReception",
+                "DetectedObjectInteraction"
```

Comment:
Does CarmaCloud care about either of these interactions. What interactions does CARMA Cloud handle.